wallet/createtx.go

```diff
@@ -1954,31 +1949,30 @@ func (w *Wallet) findEligibleOutputsAmount(dbtx walletdb.ReadTx, account uint32,
 		// manually included in transactions and sent (for example,
 		// using createrawtransaction, signrawtransaction, and
 		// sendrawtransaction).
-		class, addrs, _, err := txscript.ExtractPkScriptAddrs(
-			0, output.PkScript, w.chainParams, true) // Yes treasury
-		if err != nil || len(addrs) != 1 {
+		class, addrs := stdscript.ExtractAddrs(0 /* TODO */, output.PkScript, w.chainParams)
+		if len(addrs) != 1 {
 			continue
 		}
 
 		// Make sure everything we're trying to spend is actually mature.
 		switch class {
-		case txscript.StakeGenTy:
+		case stdscript.STStakeGenPubKeyHash, stdscript.STStakeGenScriptHash:
 			if !coinbaseMatured(w.chainParams, output.Height, currentHeight) {
 				continue
 			}
-		case txscript.StakeRevocationTy:
+		case stdscript.STStakeRevocationPubKeyHash, stdscript.STStakeRevocationScriptHash:
 			if !coinbaseMatured(w.chainParams, output.Height, currentHeight) {
 				continue
 			}
-		case txscript.TreasuryAddTy, txscript.TreasuryGenTy:
+		case stdscript.STTreasuryAdd, stdscript.STTreasuryGenPubKeyHash, stdscript.STTreasuryGenScriptHash:
 			if !coinbaseMatured(w.chainParams, output.Height, currentHeight) {
 				continue
```

Comment:
AFAICT, these three should/could be the same merged `case`, but I'm trying to minimize extraneous changes.